modules/registry-common-terms.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * registry/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-registry-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+authentication::
```

Comment:
Should we use registry authentication here?
```
OpenShift Container Platform can communicate with registries to access private image repositories using credentials supplied by the user. This allows OpenShift Container Platform to push and pull images to and from private repositories
```

Reply:
Addressed